lib/skippy/library_manager.rb

```diff
@@ -5,6 +5,7 @@
 require 'naturally'
 require 'pathname'
 require 'set'
+require 'sorted_set'
```

Comment:
Wouldn't these need a conditional loading if running under Ruby 2.x? The `Gemfile` specify the `sorted_set` gem only for Ruby 3.x.

Something like:

```rb
require 'sorted_set` if RUBY_VERSION.to_i > 2
```